src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -378,19 +380,21 @@ async def set_tpz_display_settings(self, intensity):
         payload = st.pack("<H", intensity)
         await self.send(Message(MGMSG.PZ_SET_TPZ_DISPSETTINGS, data=payload))
 
-    async def get_tpz_display_settings(self):
+    async def get_tpz_display_settings(self) -> int:
         """Get the intensity of the LED display on the front of the TPZ unit.
 
         :return: The intensity as a value from 0 (Off) to 255 (brightest).
         :rtype: int
         """
         get_msg = await self.send_request(
-            MGMSG.PZ_REQ_TPZ_DISPSETTINGS, [MGMSG.PZ_GET_TPZ_DISPSETTINGS], 1
+            MGMSG.PZ_REQ_TPZ_DISPSETTINGS, [MGMSG.PZ_GET_TPZ_DISPSETTINGS], _CHANNEL
```

Comment:
```suggestion
            MGMSG.PZ_REQ_TPZ_DISPSETTINGS, [MGMSG.PZ_GET_TPZ_DISPSETTINGS], Tpz._CHANNEL
```
Use class constants.